devp2p/kademlia.py

```diff
@@ -52,6 +53,12 @@ def __init__(self, pubkey):
         else:
             assert k_id_size == 256
             self.id = big_endian_to_int(sha3(pubkey))
+        self.ping_recv = False
+        self.pong_recv = False
+
+    @property
+    def bonded(self):
+        return self.ping_recv and self.pong_recv
```

Comment:
if a simple pickle works for the data structure, and we already have infrastructure in devp2p to read config from a specified path, i'd say yes.